src/main/java/com/coherent/token/ReadToken.java

```diff
@@ -0,0 +1,20 @@
+package com.coherent.token;
+
+import com.coherent.HttpClientFactory;
+import helpers.JsonParser;
+import org.apache.http.impl.client.CloseableHttpClient;
+
+public class ReadToken {
```

Comment:
Please check, but looks like this one isn't needed

Reply:
Removed